modules/sdk-core/src/bitgo/wallet/iWallet.ts

```diff
@@ -485,6 +485,35 @@ export interface ShareWalletOptions {
   disableEmail?: boolean;
 }
 
+export interface BulkCreateShareOptions {
+  shareOptions: Array<{
+    user: string;
+    permissions: string;
+    keychain: {
+      pub: string;
+      encryptedPrv: string;
+      fromPubKey: string;
+      toPubKey: string;
+      path: string;
+    };
+  }>;
+}
```

Comment:
```suggestion
export interface BulkKeyShareOption {
    user: string;
    permissions: string;
    keychain: {
      pub: string;
      encryptedPrv: string;
      fromPubKey: string;
      toPubKey: string;
      path: string;
    };
}
```
I dont see why it needs to be nested under `shareOptions`. Just use an array type where its being used

Reply:
The shareOptions array has been included to keep in sync with the input format for the createBulkWalletShares endpoint - https://github.com/BitGo/wallet-platform-types/pull/503/files#diff-9c2bf6507afd36913d5840997df501241100ca0db7ee94d1a5cee763d93cfbb0

Reply:
Then please lets change the createBulkWalletShares endpoint because it doesnt make any sense to nest it.

Reply:
The link isnt working. Either way I'm not sure I follow. Why do u need an unecessary nesting to accomodate additonal properties in the future? Can you give me an example? Whoever uses the interface can still use it as an array

Reply:
I think this will be a good example - https://github.com/BitGo/BitGoJS/pull/4884/files#diff-d998f4b5c3a58f8f1ac3c8075b65be2eef631b7e951bb9af3fa81ccf4af52642R496-R504 
This is the reason I preferred to keep the nesting. For ex, here we are adding a walletPassphrase along with the array. The nesting will help in adding  additional properties to the same interface.